addons/website_blog/controllers/main.py

```diff
@@ -5,14 +5,11 @@
 import urllib
 import werkzeug
 
-from openerp import tools
+from openerp import models, tools, _
 from openerp.addons.web import http
 from openerp.addons.web.http import request
 from openerp.addons.website.models.website import slug, unslug
 from openerp.exceptions import UserError
-from openerp.osv.orm import browse_record
-from openerp.tools.translate import _
-from openerp import SUPERUSER_ID
 from openerp.tools import html2plaintext
```

Comment:
organize import lines in alphabetical order like

imports of external python lib
import openerp
imports of Odoo Modules